app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,47 @@
+class QuestionsController < ApplicationController
+  before_action :find_test, only: %i[index show create destroy]
+
+  rescue_from ActiveRecord::RecordNotFound,  with: :rescue_with_question_not_found
+
+  def index
+    @questions = @test.questions.order(created_at: :asc).pluck(:body)
+    render plain: @questions * "\n"
+  end
+
+  def show
+    @question = Question.find(params[:id])
+    render plain: @question.body
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @question = Question.new(question_params)
+    if @question.save
+      redirect_to action: :show
+    else
+      render action: :new, flash: {alert: 'Some error occured.'}
```

Comment:
```suggestion
      render :new
```
обычно так пишут
такая запись повторно отрендерит страничку с формой